src/bindings/python/src/pyopenvino/utils/utils.cpp

```diff
@@ -568,6 +569,21 @@ std::shared_ptr<py::function> wrap_pyfunction(py::function f_callback) {
     });
     return callback_sp;
 }
+
+std::filesystem::path to_fs_path(const py::object& obj) {
```

Comment:
```suggestion
std::filesystem::path to_fs_path(const py::object& path) {
```
to understand what is this py::object